dev/ci/chart_schema.yaml

```diff
@@ -1,20 +1,17 @@
 #
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
+#  Copyright (c) 2025 Helm Organization.
 #
-#  http://www.apache.org/licenses/LICENSE-2.0
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
 #
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
+#      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Origin file doesn't have any header. So I add this header. Repo is maintained by Helm organization.

Reply:
IMO No header should be added and the file kept as it as the LICENSE file will contain the needed information. There is no "Helm Organisation" that owns copyright, other files in the repo have the line "Copyright The Helm Authors".

Reply:
So I should exclude these two files in the rat?